lib/cache/cache.js

```diff
@@ -146,16 +144,24 @@ class Cache {
     webidl.brandCheck(this, Cache)
     webidl.argumentLengthCheck(arguments, 1, { header: 'Cache.addAll' })
 
-    requests = webidl.converters['sequence<RequestInfo>'](requests)
```

Comment:
this is weird: any value is technically allowed in RequestInfo (it gets stringified and then parsed as a URL with the window's location typically), but undefined is very explicitly not allowed here for some reason).